.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           command: |
             . venv/bin/activate
             pip install -r requirements.txt
-            coverage run --source='authentication/,property/,transactions/,utils/,landville/' manage.py test && coverage report
+            coverage run --source='authentication/,property/,transactions/,utils/,landville/' manage.py test --keepdb && coverage report
             COVERALLS_REPO_TOKEN=${REPO_TOKEN} coveralls
```

Comment:
@Victoradukwu, The test database was destroyed each time after the test and It takes on an average 10 s to create it and to destroy it when we run the test.
Adding that argument there will help us to keep the database and gain some 10 from the time it takes to run the tests.

Reply:
`--keepdb` preserves the test database between test runs and because this is on CircleCI, it won't be practicable to respond to the prompt of creating a new db in the next test run if the db is corrupt. The `--noinput` flag takes care of that. The other alternative would be to remove the `--keepdb` flag.

Reply:
Done, I have removed it